include/eve/memory/soa_vector.hpp

```diff
@@ -0,0 +1,268 @@
+//==================================================================================================
```

Comment:
Memory is a bad place for this.
`algo -> memory`

This depends on algo.

Reply:
We decided to have `eve/container`